common.sh

```diff
@@ -457,8 +457,8 @@ if [[ ! -z ${AGENT_E2E_TEST_SCENARIO} ]]; then
 fi
 
 if [[ ! -z ${AGENT_E2E_TEST_BOOT_MODE} ]]; then
-  if [[ $AGENT_E2E_TEST_BOOT_MODE != "ISO" && $AGENT_E2E_TEST_BOOT_MODE != "PXE" ]]; then
-      printf "Found invalid value \"$AGENT_E2E_TEST_BOOT_MODE\" for AGENT_E2E_TEST_BOOT_MODE. Supported values: ISO (default), PXE."
+  if [[ $AGENT_E2E_TEST_BOOT_MODE != "ISO" && $AGENT_E2E_TEST_BOOT_MODE != "PXE" && $AGENT_E2E_TEST_BOOT_MODE != "DISKIMAGE" ]]; then
+      printf "Found invalid value \"$AGENT_E2E_TEST_BOOT_MODE\" for AGENT_E2E_TEST_BOOT_MODE. Supported values: ISO (default), PXE, DISKIMAGE."
```

Comment:
nit: feeling that a case may be a little bit more readable with a longer list, ie something like

```
case "$AGENT_E2E_TEST_BOOT_MODE" in
  "ISO" | "PXE" | "DISKIMAGE")
    # Valid value
    ;;
  *)
    printf "Found invalid value \"$AGENT_E2E_TEST_BOOT_MODE\" for AGENT_E2E_TEST_BOOT_MODE. Supported values: ISO (default), PXE, DISKIMAGE."
    exit 1

esac
```